nextjs/components/Pages/Feed/Filters/index.tsx

```diff
@@ -55,7 +58,9 @@ export default function Filters({
           ]}
           onChange={(value: ThreadState) => onChange('state', value)}
         />
-        <ScopeSelect onChange={onChange} defaultValue={defaultScope} />
+        {!!permissions.manage && (
```

Comment:
no need for `!!` here

Reply:
sometimes react renders "false" or "0", doing the !! avoids it